app/Main.hs

```diff
@@ -148,20 +160,36 @@ main :: IO ()
 main = do
   env <- getEnvironment
   opts <- execParser (optionsInfo env)
+  manager <- getManager opts
 
-  eres <- runExceptT $ runReaderT vaultEnv (env, opts)
+  eres <- runExceptT $ runReaderT vaultEnv (ExecCtx env opts manager)
   case eres of
     Left err -> hPutStrLn stderr (vaultErrorLogMessage err)
     Right newEnv -> runCommand opts newEnv
 
+getManager :: Options -> IO Manager
```

Comment:
Please add a docstring for this function. It would be nice in general to start using Haddock.
Perhaps we can open another issue for that.

Also:
s/getManager/getHttpManager

Even the type doesn't tell you anything: `IO Manager` -> I guess it manages syscalls, perhaps?

Reply:
Fixed